README.md

```diff
@@ -94,9 +94,7 @@ If you do not want to install SnowSQL, you can add Snowflake account credentials
 
 1. In your home directory, make `.snowsql` directory with a `config` file:
 
-   `mkdir .snowsql`
-   `cd .snowsql`
-   `touch config`
+   `touch ~/.snowsql/config`
```

Comment:
Think we still need the mkdir; touch won't create a directory if it doesn't exist. At least, not the version on OSX from my testing:

```shell
λ touch /tmp/some/long/path/test.txt
touch: /tmp/some/long/path/test.txt: No such file or directory

λ touch /tmp/some/path/test.txt 
touch: /tmp/some/path/test.txt: No such file or directory

λ touch /tmp/path/test.txt 
touch: /tmp/path/test.txt: No such file or directory
```

Reply:
Can we just swap it to use `mkdir ~/.snowsql && touch ~/.snowsql/config` in that case?

I probably had already created it and tried to break it down just to use the `touch` with the parent directory being present as you are correct, I don't think it would create the directory. 

In my experience, I find the fewer steps to get people started, even if it's just one or two fewer commands, makes the library seem more "digestable".

Reply:
Yeah for sure I think that would be good. Might need `mkdir -p` to make sure it doesn't return non-zero if the dir exists.